Makefile.include

```diff
@@ -290,6 +290,13 @@ term: $(filter flash, $(MAKECMDGOALS))
 doc:
 	make -BC $(RIOTBASE) doc
 
+btest:
```

Comment:
I dont know, perhaps it is better to reject this patch and eventually fill a separate PR if this approach clearly add value to test regression tools.